js/models/Profile.js

```diff
@@ -84,17 +54,23 @@ export default class extends BaseModel {
     colorFields.forEach((colorField) => {
       const clr = attrs[colorField];
 
-      if (typeof clr !== 'undefined' && is.not.hexColor(clr)) {
-        addError(colorField, 'Please provide a valid hex color.');
+      if (is.not.hexColor(clr)) {
+        addError(colorField, app.polyglot.t('profileModelErrors.provideValidHexColor'));
+      } else if (clr.charAt(0) !== '#') {
+        addError(colorField, 'The color should start with a leading hash.');
       }
     });
 
     if (attrs.email && is.not.email(attrs.email)) {
-      addError('email', 'Please provide a valid email.');
+      addError('email', app.polyglot.t('profileModelErrors.provideValidEmail'));
     }
 
     if (attrs.website && is.not.url(attrs.website)) {
-      addError('website', 'Please provide a valid url.');
+      addError('website', app.polyglot.t('profileModelErrors.provideValidURL'));
+    }
+
+    if (attrs.handle && attrs.handle.charAt(0) === '@') {
+      addError('handle', 'The handle should not start with a leading hash.');
```

Comment:
Don't forget to translate this phase. You probably want "@" instead of hash in the error.